src/containers/LearningResourcePage/components/LearningResourceTaxonomy.jsx

```diff
@@ -413,7 +421,7 @@ class LearningResourceTaxonomy extends Component {
           />
         )}
         <ResourceTypeSelect
-          availableResourceTypes={availableResourceTypes}
+          availableResourceTypes={filteredResourceTypes}
```

Comment:
Det må fremdeles være mogeleg å søke opp læringsstier for å legge disse inn i strukturen, så du kan ikkje fjerne dette heilt.

Reply:
Nei, ResourceTypeSelect komponenten filtrerte ut læringsstier. Jeg fjernet filtreringen fra komponenten, og flyttet den ut til komponenten som bruker den i steden slik at jeg kunne bruke ResourceTypeSelect uten å ha læringsstier filtrert